src/meta/processors/indexMan/CreateTagIndexProcessor.cpp

```diff
@@ -43,6 +43,37 @@ void CreateTagIndexProcessor::process(const cpp2::CreateTagIndexReq& req) {
         return;
     }
 
+    auto prefix = MetaServiceUtils::indexPrefix(space);
+    std::unique_ptr<kvstore::KVIterator> checkIter;
+    auto checkRet = kvstore_->prefix(kDefaultSpaceId, kDefaultPartId, prefix, &checkIter);
+    if (checkRet != kvstore::ResultCode::SUCCEEDED) {
+        resp_.set_code(to(checkRet));
+        onFinished();
+        return;
+    }
+
+    while (checkIter->valid()) {
+        auto val = checkIter->val();
+        auto item = MetaServiceUtils::parseIndex(val);
+        if (item.get_schema_id().getType() != nebula::cpp2::SchemaID::Type::tag_id) {
+            continue;
+        }
+
+        for (size_t i = 0; i < fieldNames.size(); i++) {
+            if (fieldNames[i] != item.get_fields()[i].get_name()) {
+                break;
+            }
+
+            if (i == fieldNames.size() - 1) {
+                LOG(ERROR) << "Tag Index " << indexName << " have existed";
+                resp_.set_code(cpp2::ErrorCode::E_EXISTED);
+                onFinished();
+                return;
+            }
+        }
+        checkIter->next();
+    }
```

Comment:
For example , exist index1 (col1, col2), then we want create index2(col1, col2, col4), is it work well?

Reply:
Sure. 

In the test case `single_index` create on `name` and `multi_index` indexed on `name` and `email`.  It's work.